lib/intl.ts

```diff
@@ -31,7 +41,7 @@ const CAL_ID = Symbol('calendar-id');
 const LOCALE = Symbol('locale');
 const OPTIONS = Symbol('options');
 
-const descriptor = (value) => {
+const descriptor = <T extends (...args: any[]) => any>(value: T) => {
```

Comment:
Could these be unknown instead of any?

Reply:
If it doens't "just work" then let's just leave it and I'll take a look some other time. Unknown is the 'better' "this could be anything" because it requires actually type-narrowing unknown before using it (instead of turning type-checking off entirely, which is what `any` does), but it can certainly be harder to use as a result.